app/controllers/collection_items_controller.rb

```diff
@@ -1,9 +1,9 @@
 class CollectionItemsController < ApplicationController
   before_action :load_collection
   before_action :load_user, only: [:update_multiple]
-  before_action :load_item_and_collection, only: [:destroy]
+  before_action :load_item_and_collection, only: [:update_multiple]
   before_action :load_collectible_item, only: [:new, :create]
-  before_action :allowed_to_destroy, only: [:destroy]
+  before_action :allowed_to_destroy, only: [:update_multiple]
```

Comment:
I'm not sure about this line, should I delete it altogether? Because it would prompt the allowed_to_destroy everytime and :update_multiple in this Collection Items is triggered. Not sure if that's the intended behavior.